### PR TITLE
Add default message for TApplicationException 

### PR DESCRIPTION
**Pull request by damnever**
Since some implementations may not set message field (field 1).

### CODE REVIEW

File: lib/go/thrift/application_exception.go [hunk 1]
```diff
@@ -30,6 +30,17 @@ const (
 	PROTOCOL_ERROR                 = 7
 )
 
+var defaultApplicationExceptionMessage = map[int32]string{
+	UNKNOWN_APPLICATION_EXCEPTION:  "unknown application exception",
+	UNKNOWN_METHOD:                 "unknown method",
+	INVALID_MESSAGE_TYPE_EXCEPTION: "invalid message type",
+	WRONG_METHOD_NAME:              "wrong method name",
+	BAD_SEQUENCE_ID:                "bad sequnce ID",
```
Comment (jeking3) [inline lib/go/thrift/application_exception.go]: Typo in the string

**Comment by jeking3** [comment]
Would it perhaps make sense to change the const list into something that can simply be generated using stringer, and therefore it would always be automatic?

https://godoc.org/golang.org/x/tools/cmd/stringer

**Comment by damnever** [comment]
Type alias is not comparable, would it cause incompatibility problems?

**Comment by jeking3** [comment]
In order to add this I need an Apache Jira ticket under the thrift project.  Do you open one?

https://thrift.apache.org/docs/HowToContribute

**Comment by damnever** [comment]
https://issues.apache.org/jira/browse/THRIFT-4315

**Pull request closed without merging**